src/models/language/languageCollection.ts

```diff
@@ -225,6 +225,7 @@ export interface ILanguageCollection extends INativeLanguageCollection {
   tuc: ILanguage;
   twig: ILanguage;
   typo3: ILanguage;
+  vapor: ILanguage;
```

Comment:
Note that language is only needed if you want to add language support. This basically means that if there's an extension in the marketplace supporting vapor, we would use it's configuration in order to decide which extensions to use.

Reply:
Oh ok, then I'll remove it on the next PR, didn't know why was this required, just copy one of the PRs.